lib/rialto/etl/configs/stanford_people_to_sparql_statements.rb

```diff
@@ -21,11 +28,13 @@
 to_field '@graph', literal(STANFORD_PEOPLE_GRAPH.to_s), single: true
 
 # Subject
-to_field '@id', extract_json('$.profileId'), single: true
-to_field '@id_ns', literal(RIALTO_PEOPLE.to_s), single: true
+# to_field '@id', extract_json('$.profileId'), single: true
```

Comment:
Do you think we should hang on to this commented-out code?

Reply:
Grrr ... I thought I double checked for legacy code. Thanks for catching.